CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 
 ### Changed
 
+- **(breaking)** - [#638](https://github.com/embedded-graphics/embedded-graphics/pull/638) Bump all crate Minimum Supported Rust Versions (MSRV) to 1.57.
```

Comment:
I think this is mistakenly under 0.7.1 instead of Unreleased.

Reply:
Good catch, fixed that. It also reminded me that I need to add the same item to `core/CHANGELOG.md` which is also done.